app/models/async_transaction/va_profile/base.rb

```diff
@@ -42,10 +42,13 @@ def self.last_requested_for_user(user)
       #   be it Email, Address, etc.
       #
       def self.start(user, response)
+        # vet360_id is no longer required for Contact Information API V2
+        source_id = Flipper.enabled?(:va_v3_contact_information_service, user) ? user.vet360_id || 0 : user.vet360_id
```

Comment:
vet360_id can be set to 0, according to the Swagger documentation

Reply:
vet360_id can be set to 0 only on V2?